src/components/popup/details/details.jsx

```diff
@@ -133,6 +139,9 @@ export default class Details extends Component {
 						/>
 						<Vendors
 							selectedVendorIds={vendorConsents}
+							selectedLegitimateInterestsIds={vendorLegitimateInterests}
```

Comment:
Maybe we could change the name of this props to selectedVendorsLegitimateInterestsIds because it is confusing inside vendors component.